README.md

```diff
@@ -1 +1,29 @@
-# hw01
\ No newline at end of file
+# Welcome ! 
+My name is **Mira** Joseph and I'm a second year Sociology/Literature student in the College. 
+
+### Interests
+
+I am interested in urban walkability, cool maps, Frederic Nietzsche, and Marx; for my BA in  sociology I am considering specializing in either social stratification or the sociology of disaster. During this period of quarantine, I am at home in ~~the greater~~ Boston ~~area.~~
+
+### What I Do
+
+In my free time I enjoy
+
+* [random wikipedia page generator](https://en.wikipedia.org/wiki/Special:Random)
+
+
+* wandering the woods with Rocky: 
+
+![](https://user-images.githubusercontent.com/63310406/78950574-2eb09500-7a9d-11ea-938e-5aa80d70e9e4.jpeg)
+
+
+* ~~being nocturnal~~ 
+
+* getting a progressively more ~~unpleasant~~ _strange_ music taste thanks to spotify's wacky algorithms 
```

Comment:
an asterisk does the same thing as the underscore function just for added knowledge.
`_strange_` is also `*strange*`